packages/pages/src/vite-plugin/plugin.ts

```diff
@@ -47,7 +47,6 @@ const cleanup = async (projectStructure: ProjectStructure): Promise<Plugin> => {
     name: "client-hydration:cleanup",
     apply: "build",
     closeBundle: {
-      order: "post",
```

Comment:
Can't we remove "pre" from the other plugin?

Reply:
Just tested and yes. Removed. 

[Sequential](https://rollupjs.org/plugin-development/): If several plugins implement this hook, all of them will be run in the specified plugin order. If a hook is async, subsequent hooks of this kind will wait until the current hook is resolved.